rm/src/main/java/io/seata/rm/DefaultResourceManager.java

```diff
@@ -121,7 +121,7 @@ public void unregisterResource(Resource resource) {
 
     @Override
     public Map<String, Resource> getManagedResources() {
-        Map<String, Resource> allResource = new HashMap<>();
+        Map<String, Resource> allResource = new HashMap<>(10);
```

Comment:
I think here keep it origin would be better.

Reply:
ok